From b1712462f87d9117eeaf433cc490d69871da6807 Mon Sep 17 00:00:00 2001
From: Jose Abreu <Jose.Abreu@synopsys.com>
Date: Thu, 14 Nov 2019 12:42:49 +0100
Subject: [PATCH] net: stmmac: Tune-up default coalesce settings

commit da20245100319b54af770df707f358f2ebc3b16d upstream

Tune-up the defalt coalesce settings for optimal values. This gives the
best performance in most of the use-cases.

Signed-off-by: Jose Abreu <Jose.Abreu@synopsys.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
[HqBai: only reset tx_frames to 25, rx_frames is not used in the version]
Signed-off-by: Haiqing Bai <Haiqing.Bai@windriver.com>
---
 drivers/net/ethernet/stmicro/stmmac/common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/common.h b/drivers/net/ethernet/stmicro/stmmac/common.h
index c265cc5770e8..ceb0d23f5041 100644
--- a/drivers/net/ethernet/stmicro/stmmac/common.h
+++ b/drivers/net/ethernet/stmicro/stmmac/common.h
@@ -251,7 +251,7 @@ struct stmmac_safety_stats {
 #define STMMAC_COAL_TX_TIMER	1000
 #define STMMAC_MAX_COAL_TX_TICK	100000
 #define STMMAC_TX_MAX_FRAMES	256
-#define STMMAC_TX_FRAMES	1
+#define STMMAC_TX_FRAMES	25
 
 /* Packets types */
 enum packets_types {
-- 
2.17.1

